docs/getting-started/libnetwork/CalicoSwarm.md

```diff
@@ -0,0 +1,127 @@
+
+    $ etcd -advertise-client-urls http://$MANAGER_IP:2379 \
+    -listen-client-urls http://0.0.0.0:2379  &
+    
+For this example, we'll only configure a single node etcd cluster on our master node. However, in a production environment, a minimum size of three nodes is recommended and can run anywhere that is accessible to the cluster.
+
+## Configure Calico
+
+Now that etcd is running, we can run Calico.
+    
+Now, start the Calico node service on each node in the cluster. 
+
+    $ sudo ETCD_AUTHORITY=$MANAGER_IP:2379 calicoctl node --libnetwork --ip=$NODE_IP
+
+## Configure Swarm Cluster
+
+Now that Calico networking is configured on our cluster, lets join our nodes into a Docker Swarm cluster.
+
+We'll use the token based discovery backend, so let's first create the token which will identify our swarm. To do this, run the following on our client. Note that docker commands may need to be run with root privileges depending on your docker installation.
+
+    $ docker pull swarm
+    $ docker run --rm swarm create
+    
+The second command will print a unique cluster ID token which we must use to configure our Swarm. We'll reference this value in the tutorial as `<swarm_token>`. Your token should look something like this:
+    d435dc104abdd89af24ae2392c7338ce
+
+## Configure Swarm Nodes
+
+Before we configure our Swarm, we will need to start the Docker daemon and point the cluster-store to our running instance of etcd. 
+
+    $ sudo docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-store=etcd://$MANAGER_IP:2379 &
+
+Now that we've got a token and a properly running daemon, we can begin to configure our Swarm. First, let's join each of our Swarm nodes to the cluster.
+
+Run the following commands on each node, replacing `<swarm_token>` with your given token.
+    
+    $ docker run -d swarm join --addr=$NODE_IP:2375 token://<swarm_token>
+    
+## Configure Swarm Manager
+
+Let's now configure the Swarm manager. Note that `<swarm_port>` in the following command can be any unused TCP port on the manager server. This is the port the client will use to communicate with the Swarm manager daemon. 
+
+    $ docker run -d -p <swarm_port>:2375 swarm manage token://<swarm_token>
+    
+## Testing Docker Swarm
+    
+At this point we should have a fully configured Calico networked Swarm cluster. However, there are no workloads running on our cluster. Let's create a few containers and check their connectivity. We can run the following commands on the client against the Swarm Manager using the -H flag.
+
+First, set the SWARM_PORT environment variable on the client to the value chosen when configuring the Swarm manager.
+
+    $ export SWARM_PORT=<swarm_port>
+       
+Then, we will configure three networks using Calico IPAM. For Calico IPAM in a non-cloud environment, you need to first create a Calico IP Pool with no additional options. Here we create a pool with CIDR 192.168.0.0/16.
```

Comment:
I'd get rid of the bit about cloud config.  Maybe tweak as follows:

We will configure three networks using the Calico network and Calico IPAM drivers.  Note that we need to first create a pool of allowable IP addresses for the containers, here we create a pool with CIDR 192.168.0.0/16: